lib/3scale/backend/stats/commons.rb

```diff
@@ -0,0 +1,41 @@
+module ThreeScale
+  module Backend
+    module Stats
+      module Commons
```

Comment:
I think that having a module for these constants makes sense :+1:  Have you considered separating the ones related with granularities from the ones that have to do with response codes?

Reply:
So far `Stats::Commons` module is small. Could be done but all belong to the `Stats::Commons` family. 

Let's keep it in a single commons module and can be split up later

Reply:
Agree to disagree.
I think this kind of modules (Commons, Utils, etc.) tend to grow as a place to put almost anything.

In this case, there are 2 clearly separate entities (granularities and response codes) and they're kind of central entities in the context of Apisonator statistics/usage, so I think it would be better to place them in different modules. No need to do this for a first version if there's not enough time, but I'd keep this in mind as a future improvement.

Reply:
Will update :+1: